examples/src/pages/Viewer.tsx

```diff
@@ -76,18 +75,7 @@ export function Viewer() {
         window.history.pushState({}, '', url.toString());
       }
 
-      let client: CogniteClient;
-      if (project && overrideToken) {
-        client = createSDKFromToken('reveal.example.example', project, overrideToken);
-      } else if (project && environment) {
-        client = await createSDKFromEnvironment('reveal.example.example', project, environment);
-      } else {
-        client = new CogniteClient({
-          appId: 'reveal.example.example',
-          project: 'dummy',
-          getToken: async () => 'dummy'
-        });
-      }
```

Comment:
I think it is somewhat smelly first create a mutable variable and then assign it immediately below with an `if` / `else if` / `else` block. If there are just two cases, I would usually go for a ternary operator inline, otherwise, I think it's better to extract a method.